test/compilable/crlf.sh

```diff
@@ -2,6 +2,8 @@
 
 # Test CRLF and mixed line ending handling in D lexer.
 
+set -e
```

Comment:
👍

I'd go for the whole `-euo pipefail` trio. Variables that might be unset should be "declared" at the top of the file, i.e. replace unset variables with empty ones.

Reply:
Yes, my plane is to enable the entire `-euo pipefail` trio by default for all bash scripts in `sh_do_test.sh`.
This is just an intermediate PR that should lay the ground work for this.
I just added `-e` here as it's an almost hard-coded reaction whenever I see `|| exit 1`